### DISCUSSION TITLE
DEV SESSION 12 Aug --- 2000 UTC+8/1400 CET/1200 GMT/0800 EDT

### DISCUSSION THREAD

**Discussion by tpike3** [mesa/mesa#1750] [General]
Where: [via Matrix](https://matrix.to/#/#mesa-dev:matrix.org)

Proposed Agenda:

- Small Development Grant and Visualization Improvements
- Mesa-Geo
- Expand the ecosystem (mesa-examples and mesa-viz-tornado) 
- Convert all `.rst` to `.md`
- CSSSA Tutorial

**Comment by tpike3** [comment]
Updated agenda to account for need build additional pypi packages mesa-examples and mesa-viz-tornado #1746 ; #1752

**Reply by tpike3** [reply]
Update to discuss .rst to .md conversion

**Comment by wang-boyu** [comment]
One more item to discuss: [2024 AAG Annual Meeting](https://www.aag.org/events/aag2024/) which might be a good place to present Mesa
